src/App.jsx

```diff
@@ -2,32 +2,61 @@ import React, { Component } from 'react';
 import './style.css';
 import 'materialize-css';
 import 'materialize-css/dist/css/materialize.min.css';
+import { loadMovies } from './actions/actionCreator';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 import SearchPage from './components/SearchPage';
 import MoviePage from './components/MoviePage';
-import axios from 'axios';
 
 class App extends Component {
-  state = {
-    data: [],
+
+  async componentDidMount() {
+    await this.props.loadMovies();
   };
 
-  refreshResults = (url) => {
-    return axios.get(url)
-      .then(response => {
-        this.setState({
-          data: response.data.data
-        });
-      });
-  }
+  moviesSorted = (movies, sortby) => {
+    const sortbyDateOrRating = {
+      'DATE': 'release_date',
+      'RATING': 'vote_average'
+    }[sortby];
+
+    const dataForSort = (data) => {
+      if (sortbyDateOrRating === 'release_date') {
+        return data.split('-').join('');
+      } else {
+        return data;
+      }
+    };
+
+    return movies.sort((a, b) => {
+      return dataForSort(b[sortbyDateOrRating]) - dataForSort(a[sortbyDateOrRating]);
+    });
+  };
 
   render() {
     return (
       <React.Fragment>
-        <SearchPage movies={this.state.data} refreshResults={this.refreshResults} />
+        <SearchPage movies={this.moviesSorted(this.props.movies, this.props.sortby)}
+          searchby={this.props.searchby}
+          sortby={this.props.sortby} />
         {/* <MoviePage movies={this.state.data}/> */}
       </React.Fragment>
     );
   }
 }
 
-export default App;
+const mapDispatchToProps = (dispatch) => {
+  return bindActionCreators({
+    loadMovies
+  }, dispatch);
+}
+
+const mapStateToProps = (state) => {
+  return {
+    movies: state.movies.data,
```

Comment:
Currently, `movies` prop is passed down through several components: App -> SearchPage -> Content
This can be avoided by using `connect` and `mapStateToProps` direclty on `Content` component.

This applies to other props and components as well - please review.